ui/src/app/applications/components/utils.tsx

```diff
@@ -656,13 +656,25 @@ export function renderResourceMenu(
                     {items.map((item, i) => (
                         <li
                             className={classNames('application-details__action-menu', {disabled: item.disabled})}
+                            tabIndex={0}
```

Comment:
Why is it always 0? Should it be `i`?

Reply:
`tabindex="0"` means that the element should be focusable in sequential keyboard navigation, after any positive tabindex values. The focus navigation order of these elements is defined by their order in the document source.